examples/launcher/src/App.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import './css/App.css';
 import Engine from './components/Engine.jsx'
 
+document.oncontextmenu = function() {
```

Comment:
I was thinking `onContextMenu={() => false}` in the JSX.

We just need to test whether that works as advertised.

Reply:
Testing now.

Reply:
moved in cfe0616